OKR/GO_CORE.md

```diff
@@ -2,7 +2,38 @@
 
 We frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
-## 2018 Q4
+## 2019 Q1
+
+### Data transfer is fast and efficient
+* It takes less than 48 hours to transfer 1TB dataset over Fast Ethernet (100Mbps)
+* Bitswap refactor + improvements reduce number of duplicate blocks downloaded by 75%
+* Transfers of sparsely provided data are roughly as efficient as fully provided content
+
+### Adding and providing large data sets works efficiently
+* Create Provider Strategies system that is solely responsible for providing
+* Implement provider strategy such that a user can add (and provide) npm or tr-wikipedia without turning off providing and without significantly impacting finding content
+
+### Datastore works efficiently for large data sets
+* Complete development of benchmarks to test speed and reliability across a wide range of datastore sizes 
+* A new datastore implementation, suitable for very large data volumes, is adopted as default in go-ipfs
+ 
+### go-ipfs becomes a well maintained project
+* Every non-trivial PR is first reviewed by someone other than @Stebalien.
+* A work-tracking process is adopted that allows the team to easily track the highest priority unclaimed issues, see the status of in progress work, and know which PRs need review, easily. 
```

Comment:
> I'd like to make this push after we get the CoreAPI in place and after we get the new RPC system. Basically, I'd like to avoid spending too much time documenting interfaces that are going to be deprecated.

@momack2 convinced me that *ignoring* documentation is a bad idea (ok, fine...) and suggested

* Every file with a non-trivial change gets fully documented.

Yes, this is going to be hard, but I think that's the only way we're going to make this happen.

Reply:
🙌 this allows us to improve and update our documentation incrementally as we touch/grok the code in service of our Q1 efforts. To measure this, we can pull all changed files in Q1 and check documentation coverage (and maybe spot-check documentation quality to ensure it's above par). Our goal with updating the docs is to make it easier for other contributors (new or old) to quickly onboard on modules and help move them forward.